app/design/adminhtml/default/default/template/page/header.phtml

```diff
@@ -31,21 +31,37 @@
             <input id="global_search" name="query" type="text" class="input-text" value="<?php if(!empty($query)): ?><?php echo $query ?><?php else: ?><?php echo Mage::helper('core')->quoteEscape($defSearch) ?><?php endif ?>" onfocus="if(this.value=='<?php echo Mage::helper('core')->quoteEscape($defSearch, true) ?>')this.value=''; " onblur="if(this.value=='')this.value='<?php echo $defSearch ?>';" />
             <div id="global_search_autocomplete" class="autocomplete"></div>
```

Comment:
add display: none to this this div to start with the right display condition on load of the page

<div id="global_search_autocomplete" class="autocomplete" style="display: none"></div>

Reply:
done